core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -635,6 +635,30 @@ class ZkUtils(val zkClient: ZkClient,
     zkClient.deleteRecursive(path)
   }
 
+  def subscribeDataChanges(path: String, listener: IZkDataListener): Unit = {
+    zkClient.subscribeDataChanges(path, listener)
+  }
+
+  def unsubscribeDataChanges(path: String, dataListener: IZkDataListener): Unit = {
+    zkClient.unsubscribeDataChanges(path, dataListener)
+  }
+
+  def subscribeStateChanges(listener: IZkStateListener): Unit = {
+    zkClient.subscribeStateChanges(listener)
+  }
+
+  def subscribeChildChanges(path: String, listener: IZkChildListener): java.util.List[String] = {
```

Comment:
The additional conversion is OK since `asJava` just provides a view over the underlying collection. The implementation of `ZkTopicEventListener` is weird, but since it's deprecated, we should probably just leave as is.